packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -112,7 +112,10 @@ const CFDFinancialStpRealAccountSignup = (props: TCFDFinancialStpRealAccountSign
     };
 
     const should_show_poi = () => {
-        if (jurisdiction_selected_shortcode === 'vanuatu' || jurisdiction_selected_shortcode === 'maltainvest') {
+        if (
+            jurisdiction_selected_shortcode === Jurisdiction.VANUATU ||
```

Comment:
We can refactor this as [Jurisdiction.VANUATU,Jurisdiction.MALTA_INVEST].includes(jurisdiction_selected_shortcode).